docs/wf-swarm-runtime/master.adoc

```diff
@@ -44,6 +44,12 @@ include::topics/wildfly-swarm/docs/howto/create-an-uberjar/index.adoc[leveloffse
 
 include::topics/dev-guide-mission-intro.adoc[leveloffset=+1]
 
+
+[[wf-swarm-available-missions-and-boosters]]
+== {WildFlySwarm} - Available Missions and Boosters
```

Comment:
Replace _{WildFlySwarm} - Available_ with _{WildFlySwarm}--Available._ (You should use an em dash, which is represented by two hyphens in the source, instead of a hyphen surrounded by spaces.)